app/libs/triggers/post_release/parallel_branches.rb

```diff
@@ -20,13 +20,16 @@ def call
     delegate :logger, to: Rails
 
     def create_and_merge_pr
+      return GitHub::Result.new { true } unless release.post_release_pr_required?
```

Comment:
In [Gitlab::Api](https://github.com/tramlinehq/tramline/blob/main/app/libs/installations/gitlab/api.rb#L153) we check for the diff inside the `create_pr` itself. Is that something we can do here for GH as well? Basically, push the logic further down.

Reply:
yes, we can do that. I will make this change.